en-US/firefox/products/developer-quantum.lang

```diff
@@ -210,8 +295,8 @@ Welcome to the all-new Firefox Quantum: Developer Edition
 Firefox has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
 
 
-;Firefox Devtools
-Firefox Devtools
+;;Firefox DevTools
+Firefox DevTools
```

Comment:
Uhm, any error on the errors page? Because you removed "Devtools" and added back "DevTools"